tests/common_testing/alfasim_sdk_common_testing/filled_case_descriptions.py

```diff
@@ -635,9 +635,12 @@
 FORMATION_LAYER_DESCRIPTION = case_description.FormationLayerDescription(
     name="f", start=Scalar(1, "m"), material="Carbon Steel"
 )
+FORMATION_LAYER_DESCRIPTION_1 = case_description.FormationLayerDescription(
+    name="f (1)", start=Scalar(1, "m"), material="Carbon Steel"
+)
 FORMATION_DESCRIPTION = case_description.FormationDescription(
     reference_y_coordinate=Scalar(1, "m"),
-    layers=[FORMATION_LAYER_DESCRIPTION, FORMATION_LAYER_DESCRIPTION],
+    layers=[FORMATION_LAYER_DESCRIPTION, FORMATION_LAYER_DESCRIPTION_1],
 )
```

Comment:
Just a comment.

"Carbon Steel" formation =P
Are we drilling some weird mineral deposit?